src/main.rs

```diff
@@ -228,6 +228,9 @@ fn init_logging(level: log::LevelFilter) -> Result<(), fern::InitError> {
     }
 
     if let Some(log_file) = CONFIG.log_file() {
+        if OpenOptions::new().write(true).create(true).open(&log_file).is_err() {
```

Comment:
I'd use `append(true)` instead of `write(true)`. It's technically not necessary, but it "feels" safer.

```suggestion
        if OpenOptions::new().append(true).create(true).open(&log_file).is_err() {
```

Reply:
Makes sense! Good call, committing now